.editorconfig

```diff
@@ -30,3 +35,8 @@ indent_size = 2
 [*.{yml,yaml}]
 indent_size = 2
 indent_style = space
+
+# Ignore assets
+[{images,sounds}/**]
+insert_final_newline = unset
+trim_trailing_whitespace = unset
```

Comment:
Why do we need to exclude these directories?

Reply:
It's because the checker seems to think some asset files are text files and emits a ton of errors about trailing whitespace.

Reply:
Looks like .wav and .mp3 aren't in the [default excludes](https://github.com/editorconfig-checker/editorconfig-checker#default-excludes), so we would need to add them.

PNG are definitely in the excluded list, though, so not sure which files in `images/` are causing the issue.

Reply:
Are those excludes official? Because if not some other editorconfig checker tool could try to verify images as well (but I don't know). It's seems safer to simpler ignore everything in those two folders.